### ISSUE TITLE
isocurve example does not run

### ISSUE THREAD

**Issue by michaelaye** [vispy/vispy#1012]
tried to run the isocurve_for_trisurface_qt example (using Python 3 on Macbook Air, Miniconda with all updates)

``` python
$ python isocurve_for_trisurface_qt.py
WARNING: Traceback (most recent call last):
  File "isocurve_for_trisurface_qt.py", line 128, in <module>
    win.show()
  File "/Users/klay6683/Dropbox/src/vispy_campagnola/vispy/app/backends/_qt.py", line 646, in paintGL
    self._vispy_canvas.events.draw(region=None)
  File "/Users/klay6683/Dropbox/src/vispy_campagnola/vispy/util/event.py", line 455, in __call__
    self._invoke_callback(cb, event)
  File "/Users/klay6683/Dropbox/src/vispy_campagnola/vispy/util/event.py", line 475, in _invoke_callback
    self, cb_event=(cb, event))
  << caught exception here: >>
  File "/Users/klay6683/Dropbox/src/vispy_campagnola/vispy/util/event.py", line 471, in _invoke_callback
    cb(event)
  File "/Users/klay6683/Dropbox/src/vispy_campagnola/vispy/scene/canvas.py", line 205, in on_draw
    self._draw_scene()
  File "/Users/klay6683/Dropbox/src/vispy_campagnola/vispy/scene/canvas.py", line 251, in _draw_scene
    self.draw_visual(self.scene)
  File "/Users/klay6683/Dropbox/src/vispy_campagnola/vispy/scene/canvas.py", line 289, in draw_visual
    node.draw()
TypeError: draw() missing 1 required positional argument: 'transforms'
ERROR: Invoking <bound method Canvas.on_draw of <Canvas (PyQt4) at 0x1079b9128>> for DrawEvent
ERROR: Invoking <bound method Canvas.on_draw of <Canvas (PyQt4) at 0x1079b9128>> repeat 2
```

**Comment by Eric89GXL** [comment]
It should be fixed as part of the mega-PR #928. We hope to merge it soon.

**Comment by kmuehlbauer** [comment]
Closed, example is working with latest vispy and Qt4
